patternfly/patternfly-elements
# chore: Remove globbing from pfe-sass

### What has changed and why

In the pfe-sass.scss file, the individual file imports were added from each folder rather than referencing the automatically globbed files.

Removed the globbing task from the build and the dependencies related to that task.

### Testing instructions

1. `npm run dev pfe-sass`: Should build a clean Sassdocs and open a localhost view.
2. `npm run dev pfe-cta`: Should build pfe-cta correctly, no Sass errors.


### Ready-for-merge Checklist




### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
